### PR TITLE
[4] cannot pass null to trim function in php 8.1

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue # .
cannot pass null to trim function in php 8.1

### Summary of Changes
set a default value

### Testing Instructions
go to /index.php?option=com_plugins&view=plugins
look the log 

### Actual result BEFORE applying this Pull Request
deprecation from WebApplication .php 
trim() passing null to parameter #1 ($string)

### Expected result AFTER applying this Pull Request

no deprecation from WebApplication .php

### CODE REVIEW

File: libraries/src/Application/WebApplication.php [hunk 1]
```diff
@@ -413,7 +413,7 @@ protected function loadSystemUris($requestUri = null)
 			else
 			{
 				// Normalise slashes.
-				$mediaURI = trim($mediaURI, '/\\');
+				$mediaURI = trim($mediaURI  ?? '', '/\\');
```
Comment (richard67) [inline libraries/src/Application/WebApplication.php]:
```suggestion
				$mediaURI = trim($mediaURI ?? '', '/\\');
```
Fix PHPCS for drone.

**Comment by xillibit** [comment]
I have tested this item :white_check_mark: successfully on bbd22e2205a65cee19a5d69507e18999d8979b26<br /><br />The deprecation on WebApplication.php are right disappear with this path on Php 8.1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36248">issues.joomla.org/tracker/joomla-cms/36248</a>.</sub>

**Comment by wilsonge** [comment]
Sorry I missed this one when I merged all your others yesterday but have patched this myself

**Comment by alikon** [comment]
no problem

**Pull request closed without merging**